scripts/archive.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+source  /etc/profile.d/modules.sh
+
+set -a
+
+module purge
+module load pbs
+module use /g/data/hh5/public/modules
+module use ~access/modules
+module load cdo
+module load nco
+module load pythonlib/um2netcdf4/2.0
+module load conda/analysis3
+
+# set the environment variables for ACCESS-Archiver
+loc_exp=${PWD##*/}
+subdaily=false
+access_version=esmpayu
+arch_dir=/g/data/$PROJECT/$USER/archive/access-esm
+base_dir=/scratch/$PROJECT/$USER/access-esm/archive
+comp_proj=$PROJECT
+here=/g/data/tm70/kr4383/ACCESS-Archiver 
+
+plev8=false
+zonal=false
+convert_unknown=false
+ncexists=false
+arch_grp=$PROJECT
+UMDIR=/projects/access/umdir
+
+# setup dircetories
+mkdir -p $here/tmp/$loc_exp
+rm -f $here/tmp/$loc_exp/*
+mkdir -p $arch_dir/$loc_exp/{history/atm/netCDF,restart/atm}
+
+# run access archiving scripts
+${here}/subroutines/find_files_payu.sh # create file list
+python -W ignore $here/subroutines/run_um2nc.py # convert UM files to netcdf
+$here/subroutines/cp_hist_payu.sh # copy over history
+$here/subroutines/cp_rest_payu.sh # copy over output files
+$here/subroutines/mppnccomb_check.sh # combine ocean output files
```

Comment:
I'm wondering why this is necessary. It is trying to collate ocean files that should already be collated by `payu`

Reply:
I'm using ACCESS-Archiver to do the collating rather than `payu` here. It made using the ACCESS-Archiver workflow easier, and I'm planning on adding the CMIP6 conversion to this script in another PR which needs to run after collation. I couldn't see a good way to add in a task after collation but there's probably a better way :)

Reply:
If it makes it easy then go for it. The way it collates is strange to my mind, but if it works then fine. Bear in mind that high resolution ocean experiments use [mppnccombine-fast](https://github.com/coecms/mppnccombine-fast), but that doesn't apply here. 

If you do want to run something after collation you can use [`postscript`](https://payu.readthedocs.io/en/latest/config.html#miscellaneous). Note that this is a script that is submitted via PBS. As an example the COSIMA experiments use it for synching data:

https://github.com/COSIMA/01deg_jra55_iaf/blob/master/config.yaml#L77